access/client.go

```diff
@@ -102,6 +102,15 @@ type Client interface {
 	// GetExecutionResultForBlockID gets the execution results at the block ID.
 	GetExecutionResultForBlockID(ctx context.Context, blockID flow.Identifier) (*flow.ExecutionResult, error)
 
+	// GetExecutionDataByBlockID returns execution data for a specific block ID.
+	GetExecutionDataByBlockID(ctx context.Context, blockID flow.Identifier) (*flow.ExecutionData, error)
+
+	// SubscribeExecutionData subscribes to execution data updates starting at the given block ID or height.
+	SubscribeExecutionData(ctx context.Context, startBlockID flow.Identifier, startHeight uint64) (<-chan flow.ExecutionDataStreamResponse, <-chan error, error)
```

Comment:
Might be a good idea to break this into two functions: `ByHeight` and `ById`

Reply:
@sideninja @nvdtf what's your opinion on this? Currently the rpc accepts either height/blockID which is a bit of an outlier for the Access API. All of the existing methods match the rpc.

1. Keep it consistent with the rpc (keep it as-is in this PR)
2. Keep it consistent with the other methods (add `*ByBlockID`, `*ByHeight` versions
3. Align the protobuf with 2. If we go this route, I vote we do 1, then make those changes separately.

Reply:
I think 2 is good enough. To me, accepting two fields and relying on either one is not good developer experience. Go SDK as the SDK can break this into two functions and call the RPC properly. 
I like 3 as well, but don't want to push extra work for minor improvements.

Reply:
updated in [4275139](https://github.com/onflow/flow-go-sdk/pull/417/commits/4275139b17fea0894178a181979c1a83cc4022f4)